packages/web/src/common/store/notifications/sagas.ts

```diff
@@ -547,6 +574,9 @@ export function* getNotifications(isFirstFetch: boolean) {
             withDethroned
           })
         )
+      if (notificationsResponse && 'notifications' in notificationsResponse) {
+        notificationsResponse.notifications = notifications
+      }
```

Comment:
Note there is a follow up to get the unread count and playlist update notifications from discovery as well so we won't have to rely on the initial identity call soon